docs/conf.py

```diff
@@ -93,7 +93,7 @@
 
 # The theme to use for HTML and HTML Help pages.  Major themes that come with
 # Sphinx are currently 'default' and 'sphinxdoc'.
-html_theme = 'default'
+html_theme = 'classic'
```

Comment:
Apparently sphinx has changed the name of their default. I had to make this change to get the docs build to pass.